app/models/user.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
@aetheryna Should we enable just only what we're using following the ticket detail?
For example, we don't have the `rememberable` feature now so we can remove that out 🙌

Reply:
@Lahphim Yeah, your right removed it 😄 Updated in https://github.com/aetheryna/google-scraper-ruby/pull/31/commits/d32bb263320e1633f242f39918a3f1aaf69c7c81 😎